src/scss/pages/settings/_settings.scss

```diff
@@ -3,13 +3,19 @@
 }
 
 .settings-section {
-  @include flex (column, flex-start, stretch);
-  flex-wrap: wrap;
   @extend .sheet;
-  padding: 0 $padding-medium $padding-medium !important;
+  padding: $padding-medium $padding-medium $padding-medium !important;
```

Comment:
It can just be `padding: $padding-medium` now. Is the important clause still needed?